README.md

```diff
@@ -51,31 +51,32 @@ Bellow is a basic setup example to use AMIE with a remote knowledge base.
 
```

Comment:
I think it may be a good idea to say that the client and the server use the WebSocket protocol to communicate ;-)